<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setup</class>
 <widget class="QWidget" name="Setup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_Start">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Start the simulation</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>391</width>
     <height>109</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="label_SOD">
        <property name="toolTip">
         <string>Source to object distance</string>
        </property>
        <property name="text">
         <string>Source to Object Distance (mm) :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_SdD">
        <property name="toolTip">
         <string>Source to diffuser distance</string>
        </property>
        <property name="text">
         <string>Source to Diffuser Distance (mm) :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_SDD">
        <property name="toolTip">
         <string>Source to detector distance</string>
        </property>
        <property name="text">
         <string>Source to Detector Distance  (mm):</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEdit_SOD"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_SdD"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_SDD"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>315</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_NumProj">
      <property name="toolTip">
       <string>Number of Projections</string>
      </property>
      <property name="text">
       <string>Number of Projections : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_NumProj"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>51</y>
     <width>386</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>160</number>
    </property>
    <item>
     <widget class="QRadioButton" name="radioButton_Radiography">
      <property name="toolTip">
       <string>Radiography</string>
      </property>
      <property name="text">
       <string>Radiography</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_Tomography">
      <property name="toolTip">
       <string>Tomography</string>
      </property>
      <property name="text">
       <string>Tomography</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
